runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -580,6 +584,17 @@ public void stopLoading() {
      * @return the url for current web page/app.
      * @since 1.0
      */
+    
+    @XWalkAPI
```

Comment:
Above comment is for `getUrl`, so please adjust the order and add comment for `getHitTestResult`.